src/backend/api/Fusion.Resources.Domain/Queries/GetPersonAbsence.cs

```diff
@@ -32,9 +32,8 @@ public async Task<IEnumerable<QueryPersonAbsence>> Handle(GetPersonAbsence reque
                 var items = await db.PersonAbsences.GetById(request.PersonId)
                     .Include(x => x.Person)
                     .Include(x => x.CreatedBy)
-                    .ToListAsync();
-
-
+                    .Include(x => x.TaskDetails)
+                    .ToListAsync(cancellationToken: cancellationToken);
```

Comment:
do you have to specify the cancellationToken param? Would rather just pass it as the name is descriptive enough..